flake.nix

```
@@ -2,8 +2,8 @@
   description = "A flake for getting started with Scala.";
 
   inputs = {
-    nixpkgs.url = github:nixos/nixpkgs/nixpkgs-unstable;
-    flake-utils.url = github:numtide/flake-utils;
+    nixpkgs.url = "github:kubukoz/nixpkgs/scala-cli-move-assert";
```

This can be updated to

```suggestion
    nixpkgs.url = "github:nixos/nixpkgs";
```

after this PR is merged: https://github.com/NixOS/nixpkgs/pull/208145

---

(it'll require running a flake-enabled command so that the lockfile is also updated)